x-pack/test/functional_enterprise_search/with_host_configured.config.ts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Every time you run a test with the functional test runner, you need to specify which configuration file to use.

Kibana configuration can be set in these files. Configuration like `enterpriseSearch.host`.

Because of this, we need 2 separate files for testing when `enterpriseSearch.host` is set or not, which is why we also need two test commands, as noted in the README.

Both test files extend a common "base" config file for shared concerns.